src/main/webapp/app/routes.tsx

```diff
@@ -29,6 +30,7 @@ const Routes = () => (
       <ErrorBoundaryRoute exact path="/auth/email/login" component={AuthEmailLogin} />
       <ErrorBoundaryRoute exact path="/auth/email/register" component={AuthEmailRegister} />
       <ErrorBoundaryRoute exact path="/auth/email/reset" component={AuthEmailReset} />
+      <ErrorBoundaryRoute path="/events/:id/details" component={EventDetails} />
```

Comment:
page path better to defined in synchronized place
[event/index.tsx](https://github.com/Five-Fishes/Club-Management-React-Client/blob/dev-version-1/src/main/webapp/app/entities/event/index.tsx)